libraries/stdlib/genmdl/stdlib_genmdl_impl.mtlx

```diff
@@ -1,902 +1,722 @@
-
-  <!-- <rgbtohsv> -->
-  <implementation name="IM_rgbtohsv_color3_genmdl" nodedef="ND_rgbtohsv_color3" file="stdlib/genmdl/mx_rgbtohsv_color3.osl" function="mx_rgbtohsv_color3" language="genmdl"/>
-  <implementation name="IM_rgbtohsv_color4_genmdl" nodedef="ND_rgbtohsv_color4" file="stdlib/genmdl/mx_rgbtohsv_color4.osl" function="mx_rgbtohsv_color4" language="genmdl"/>
-
-  <!-- <hsvtorgb> -->
-  <implementation name="IM_hsvtorgb_color3_genmdl" nodedef="ND_hsvtorgb_color3" file="stdlib/genmdl/mx_hsvtorgb_color3.osl" function="mx_hsvtorgb_color3" language="genmdl"/>
-  <implementation name="IM_hsvtorgb_color4_genmdl" nodedef="ND_hsvtorgb_color4" file="stdlib/genmdl/mx_hsvtorgb_color4.osl" function="mx_hsvtorgb_color4" language="genmdl"/>
-
-  <!-- ======================================================================== -->
-  <!-- Compositing nodes                                                        -->
-  <!-- ======================================================================== -->
-
-  <!-- <premult> -->
-  <implementation name="IM_premult_color2_genmdl" nodedef="ND_premult_color2" file="stdlib/genmdl/mx_premult_color2.osl" function="mx_premult_color2" language="genmdl"/>
-  <implementation name="IM_premult_color4_genmdl" nodedef="ND_premult_color4" file="stdlib/genmdl/mx_premult_color4.osl" function="mx_premult_color4" language="genmdl"/>
-
-  <!-- <unpremult> -->
-  <implementation name="IM_unpremult_color2_genmdl" nodedef="ND_unpremult_color2" file="stdlib/genmdl/mx_unpremult_color2.osl" function="mx_unpremult_color2" language="genmdl"/>
-  <implementation name="IM_unpremult_color4_genmdl" nodedef="ND_unpremult_color4" file="stdlib/genmdl/mx_unpremult_color4.osl" function="mx_unpremult_color4" language="genmdl"/>
-
-  <!-- <plus> -->
-  <implementation name="IM_plus_float_genmdl" nodedef="ND_plus_float" file="stdlib/genmdl/mx_plus.inline" language="genmdl"/>
-  <implementation name="IM_plus_color2_genmdl" nodedef="ND_plus_color2" file="stdlib/genmdl/mx_plus.inline" language="genmdl"/>
-  <implementation name="IM_plus_color3_genmdl" nodedef="ND_plus_color3" file="stdlib/genmdl/mx_plus.inline" language="genmdl"/>
-  <implementation name="IM_plus_color4_genmdl" nodedef="ND_plus_color4" file="stdlib/genmdl/mx_plus.inline" language="genmdl"/>
-
-  <!-- <minus> -->
-  <implementation name="IM_minus_float_genmdl" nodedef="ND_minus_float" file="stdlib/genmdl/mx_minus.inline" language="genmdl"/>
-  <implementation name="IM_minus_color2_genmdl" nodedef="ND_minus_color2" file="stdlib/genmdl/mx_minus.inline" language="genmdl"/>
-  <implementation name="IM_minus_color3_genmdl" nodedef="ND_minus_color3" file="stdlib/genmdl/mx_minus.inline" language="genmdl"/>
-  <implementation name="IM_minus_color4_genmdl" nodedef="ND_minus_color4" file="stdlib/genmdl/mx_minus.inline" language="genmdl"/>
-
-  <!-- <difference> -->
-  <implementation name="IM_difference_float_genmdl" nodedef="
```

Comment:
I'm not sure we want to merge this in either, as I suspect our node implementations for MDL will be using "inlining" a lot, and the current definitions demonstrates how this is done.

Reply:
This I think is worth leaving in to see the required naming associations. I was going to see about inlines.